src/libsync/theme.h

```diff
@@ -203,6 +203,7 @@ class OWNCLOUDSYNC_EXPORT Theme : public QObject
      * About dialog contents
      */
     virtual QString about() const;
+    virtual bool aboutShowCopyright() const;
```

Comment:
(I would have made this return a QString, so they can customize the name if they want (or empty to hide), I foresee that customer will want that.)

Reply:
Use `APPLICATION_VENDOR` here?

Should be `ownCloud GmbH` in the unbranded?
https://github.com/owncloud/client/blob/v2.5.0/OWNCLOUD.cmake#L5

Reply:
Yeah, even when `APPLICATION_VENDOR` is "Acme Corp LLC", copyright is still "ownCloud GmbH".